### DISCUSSION TITLE
Is there a console.log equivalent in flow?

### DISCUSSION THREAD

**Discussion by yatw** [postmanlabs/postman-flows#378] [Ideas]
We are using flow to test multiple API in a sequence. And currently the only way to see the status is click open the console and drill down to response body to see the API result. Terminal block works the same way, we cannot print custom message. I am looking for something simple `console.log("Invite User Id=5 failed") ` which we can easily do in script. 

Is there a different way to do this in flow or is this a new feature request?

**Comment by Ariemeth** [comment]
As far as I can tell that would be a feature request.  I currently handle this by adding a create data block between the request and terminal format the data before sending to a named terminal
